images/win/Windows2016-Azure.json

```diff
@@ -124,7 +124,8 @@
             ],
             "scripts":[
                 "{{ template_dir }}/scripts/Installers/Install-PowerShellModules.ps1",
-                "{{ template_dir }}/scripts/Installers/Initialize-VM.ps1"
+                "{{ template_dir }}/scripts/Installers/Initialize-VM.ps1",
+                "{{ template_dir }}/scripts/Installers/Install-WebPI.ps1"
```

Comment:
Let's name it with full name `Install-WebPlatformInstaller`

Reply:
Thank you for the suggestion, renamed to `Install-WebPlatformInstaller`